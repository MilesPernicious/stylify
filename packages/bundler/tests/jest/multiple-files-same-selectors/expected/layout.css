.k{
	border: 2px solid transparent
}
.l{
	float: left
}
.j{
	color: blue
}
.m{
	visibility: hidden
}
.g{
	display: flex
}
.n{
	-webkit-user-select: none;
	   -moz-user-select: none;
	        user-select: none
}
.h{
	justify-content: center
}

@media (min-width: 768px) {
.a{
	color: red
}
}

@media (min-width: 1024px) {
.b{
	color: green
}
}
