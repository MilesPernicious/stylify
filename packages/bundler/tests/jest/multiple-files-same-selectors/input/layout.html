<div class="border:2px__solid__transparent float:left color:blue md:color:red lg:color:green">
	<div class="visibility:hidden display:flex user-select:none justify-content:center"></div>
</div>
