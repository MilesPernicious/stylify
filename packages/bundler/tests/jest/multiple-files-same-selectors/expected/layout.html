<div class="k l j a b">
	<div class="m g n h"></div>
</div>
